### PR TITLE
Change <Fragment> to <> with minimal dependency update

### PR DESCRIPTION
**Pull request by marcin-zapala**

### CODE REVIEW

**Comment by NicolasMartos** [comment]
Good for me! Let @renaudtertrais have a look

File: .babelrc [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+    ["@babel/preset-env", {
       "targets": {
         "browsers": ["last 3 versions", "IE >= 11"]
       }
     }],
-    "react"
+    "@babel/preset-react"
   ],
-  "plugins": ["transform-object-rest-spread"]
+  "plugins": ["@babel/plugin-proposal-object-rest-spread"]
```
Comment (renaudtertrais) [inline .babelrc]: we should need this anymore

File: package.json [hunk 2]
```diff
@@ -33,21 +33,24 @@
     "start": "npm run start:storybook"
   },
   "devDependencies": {
+    "@babel/cli": "^7.4.4",
+    "@babel/core": "^7.4.5",
+    "@babel/plugin-proposal-object-rest-spread": "^7.4.4",
```
Comment (renaudtertrais) [inline package.json]: this should not be needed

File: package.json [hunk 3]
```diff
@@ -33,21 +33,24 @@
...
     "@storybook/addon-centered": "^3.4.2",
     "@storybook/addon-info": "^3.4.2",
     "@storybook/addon-jest": "^3.4.2",
     "@storybook/addon-knobs": "^3.4.2",
     "@storybook/addon-options": "^3.4.2",
-    "@storybook/addon-actions": "3.4.12",
     "@storybook/react": "^3.4.2",
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.0",
+    "babel-core": "7.0.0-bridge.0",
```
Comment (renaudtertrais) [inline package.json]: no version more "stable" ? is it the last one ?

File: package.json (hunk 2 above)
Comment (marcin-zapala) [reply package.json]: removed

File: package.json (hunk 3 above)
Comment (marcin-zapala) [reply package.json]: this is the last one and also some dependency force me to install this because:  https://github.com/babel/babel-bridge

**Pull request merged**